src/cromshell/submit/command.py

```diff
@@ -65,6 +68,11 @@ def main(config, wdl, wdl_json, options_json, dependencies_zip, no_validation):
 
     http_utils.assert_can_communicate_with_server(config=config)
 
+    if io_utils.has_nested_dependencies(wdl):
```

Comment:
It would be a good idea to have the option to disable this feature.
Also Logger.debug message saying this is taking place within the if statement